stdlib/3/time.pyi

```diff
@@ -65,3 +73,18 @@ def strptime(string: str,
              format: str = ...) -> struct_time: ...
 def time() -> float: ...
 def tzset() -> None: ...  # Unix only
+
+if sys.version_info >= (3, 3):
+    def clock_getres(int) -> float: ...
+
+    def clock_gettime(int) -> float: ...
+
+    def clock_settime(int, struct_time) -> float: ...
+
+    def get_clock_info(str) -> Dict[str, Any]: ...
```

Comment:
It really doesn't return a dict -- the docstring is just wrong. (You can file a doc bug on bugs.python.org about that!) There are two ways to go about this: either declare it as `types.SimpleNamespace` (that type exists in typeshed though its stub is feeble), or define a custom class whose name starts with `_` that has the fields documented here: https://docs.python.org/3/library/time.html#time.get_clock_info.